index.js

```diff
@@ -109,6 +109,9 @@ module.exports.api = function (settings) {
 
   router.setupCollectionDb = async function(collection) {
     debug(`init collection db ${collection._id} ${collection.storage}`)
+    if (collection.storage === 'postgres') {
```

Comment:
should probably check for existence of `router.db.pgpool` here, but it read terribly so opted not to

Reply:
Seems fine to me as is since the util caches.